.editorconfig

```diff
@@ -0,0 +1,13 @@
+
+root = true
+
+[*]
+trim_trailing_whitespace = true
+insert_final_newline = true
+end_of_line = lf
+charset = utf-8
+max_line_length = 180
```

Comment:
This is exactly right for our displays 😂

Reply:
well, I dunno about you but my monitor is split between IDE and browser. Also, needs to fit on a laptop screen.

Reply:
No, I was joking :). `180` is totally fine. I usually go into flamewars with people who insist on `80`.